jdi-light-mobile-tests/src/main/java/io/github/com/custom/ContactFormCustomGet.java

```diff
@@ -0,0 +1,31 @@
+package io.github.com.custom;
+
+import com.epam.jdi.light.elements.complex.dropdown.Dropdown;
+import com.epam.jdi.light.elements.composite.Form;
+import com.epam.jdi.light.elements.interfaces.complex.IsCombobox;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.*;
+import io.github.com.entities.Contacts;
+
+import java.lang.reflect.Field;
+
+public class ContactFormCustomGet extends Form<Contacts> {
+    TextField name;
+    TextField lastName, position, passportNumber, passportSeria;
+
+    Dropdown gender;
+    IsCombobox religion;
+
+    Checkbox passport, acceptConditions;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5217753)